src/controllers/legacycontrollersettings.cpp

```diff
@@ -230,7 +230,8 @@ LegacyControllerEnumSetting::LegacyControllerEnumSetting(
             !value.isNull();
             value = value.nextSiblingElement("value")) {
         QString val = value.text();
-        m_options.append(std::tuple<QString, QString>(val, value.attribute("label", val)));
+        QColor color = QColor(value.attribute("color"));
+        m_options.append(Item{val, value.attribute("label", val), color});
```

Comment:
```suggestion
        m_options.emplace_back(val, value.attribute("label", val), color);
```

Reply:
Looks like I still need to the `stuct` type